fontforge/sflayout.c

```diff
@@ -504,7 +504,7 @@ void LayoutInfoRefigureLines(LayoutInfo *li, int start_of_change,
     li->pcnt += pdiff;
 
     if ( li->lmax <= li->lcnt+lcnt - (le-ls) + 1 ) {
-	li->lines = realloc(li->lines,(li->lmax = li->lcnt+30+lcnt-(le-ls+1))*sizeof(struct openfont_str **));
+	li->lines = realloc(li->lines,(li->lmax = li->lcnt+30+lcnt-(le-ls+1))*sizeof(struct opentype_str **));
```

Comment:
Could you please also change this to `struct opentype_str *`?

Reply:
Did this somehow get fixed without dropping the comment?

Reply:
Thank you for your review, jtanx. 

- Of course, these two typos do not change the behavior of the program.
- To be honest,  I am not an expert in fontforge :). I found these two problems when I perform a static analysis on fontforge. I believe that changing `struct opentype_str**` to `struct opentype_str*` requires someone more familiar with fontforge. And this work can be done in the next commit. Sorry for this.

Reply:
Thank you for your comment, frank-trample.
What do you mean? Should I add comments for this change?

Reply:
@movie-travel-code there's nothing to analyse, in this context, `sizeof struct opentype_str*` is what it should be.

Reply:
@jtanx In `li->lines = realloc(ptr, length*sizeof(struct openfont_str **));`, the type of `li-lines` is **`struct opentype_str ***`**, see https://github.com/fontforge/fontforge/blob/master/fontforge/sflayoutP.h#L85, so I think `sizeof(struct opentype_str **) ` is more appropriate than `sizeof(struct opentype_str *)`. Of course, whatever it is, it doesn't matter.

Thank you again for taking the time to review the code.

Reply:
@movie-travel-code sorry, you're right. I was looking at https://github.com/fontforge/fontforge/pull/3218/files#diff-fe9fdb4418dbd4edec366102cd4f55a8R405 for reference, but that one is wrong.